_gsocprojects/2019/project_ROOT.md

```diff
@@ -0,0 +1,14 @@
+---
+project: ROOT
+layout: default
+logo: rootlogo.png
+description: |
+  [ROOT](http://root.cern.ch) is a modular scientific software framework developed by [CERN](http://cern.ch) and other Particle Physics laboratories.
+  It was originally designed for particle physics data analysis, but it is also used in other applications such as astronomy, bio-informatics, finances, etc.
+  It provides all the functionalities needed to deal with big data processing, statistical analysis, data visualisation and data storage.
+  It is mainly written in C++ but integrated with other languages such as Python and R.
+summary: |
+  A modular scientific software framework. It provides all the functionalities needed to deal with big data processing,
+  statistical analysis, visualisation and storage. It is mainly written in C++ but integrated with other languages such
+  as Python and R.
+---
```

Comment:
@yamaguchi1024 could you please add this line at the end of this file?

```

{% include gsoc_project.ext %}
```

In this case, it is needed.